modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/Parameter.java

```diff
@@ -16,92 +16,103 @@
 
   /**
    * Additional description data to provide on the purpose of the parameter
+   * @return the parameter's description
    **/
   String description() default "";
 
   /**
-   * Specifies that the parameter is not optional and must be present
+   * Specifies that the parameter is not optional and must be present.
+   * @return whether or not the parameter is required
    **/
   boolean required() default false;
 
   /**
-   * 
+   * Specifies that a parameter is deprecated and should be transitioned out of usage.
+   * @return whether or not the parameter is deprecated
    **/
   boolean deprecated() default false;
 
   /**
    * When true, allows sending an empty value.  If false, the parameter will be considered \&quot;null\&quot; if no value is present.  This may create validation errors when the parameter is required.
+   * @return whether or not the parameter allows empty values
    **/
   boolean allowEmptyValue() default false;
 
   /**
-   * 
+   * Describes how the parameter value will be serialized depending on the type of the parameter value. Default values (based on value of in): for query - form; for path - simple; for header - simple; for cookie - form.
+   * @return the style of the parameter
    **/
   String style() default "";
 
   /**
-   * 
+   * When this is true, parameter values of type array or object generate separate parameters for each value of the array or key-value pair of the map. For other types of parameters this property has no effect. When style is form, the default value is true. For all other styles, the default value is false.
+   *@return whether or not to expand individual array members
    **/
   boolean explode() default false;
 
   /**
-   * 
+   * Determines whether the parameter value should allow reserved characters, as defined by RFC3986. This property only applies to parameters with an in value of query. The default value is false.
+   * @return whether or not the parameter allows reserved characters
    **/
   boolean allowReserved() default false;
 
   /**
-   * 
+   * The schema defining the type used for the parameter.
+   * @return the schema of the parameter
    **/
   Schema schema() default @Schema();
 
-
-  ArraySchema array() default @ArraySchema();
-
-  // TODO #2312 single object, according to spec
   /**
-   * 
+   * The representation of this parameter, for different media types.
+   * @return the content of the parameter
    **/
   Content[] content() default @Content();
 
   /**
-   * allows parameter to be marked as hidden
+   * Allows this parameter to be marked as hidden
+   * @return whether or not this parameter is hidden
    */
   boolean hidden() default false;
+  
+  /**
+   * Provides an array examples of the schema.  When associated with a specific media type, the example string shall be parsed by the consumer to be treated as an object or an array.
+   * @return the list of examples for this parameter
+   **/
+  String[] examples() default {};
```

Comment:
The Parameter object in the spec has an `examples` field: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#parameterObject